types/block.go

```diff
@@ -593,9 +593,14 @@ const (
 const (
 	// Max size of commit without any commitSigs -> 82 for BlockID, 8 for Height, 4 for Round.
 	MaxCommitOverheadBytes int64 = 94
-	// Commit sig size is made up of 96 bytes for the signature, 20 bytes for the address,
-	// 1 byte for the flag and 14 bytes for the timestamp.
-	MaxCommitSigBytes int64 = 131 + 10 // where's the 10 from?
+
+	// 4 bytes for field tags + 1 byte for signature LEN + 1 byte for
+	// validator address LEN + 1 byte for timestamp LEN.
+	maxCommitSigProtoEncOverhead = 4 + 1 + 1 + 1 + 3 // 3 ???
```

Comment:
still don't know where 3 bytes are coming from

Reply:
```
message CommitSig {
  BlockIDFlag               block_id_flag     = 1;
  bytes                     validator_address = 2;
  google.protobuf.Timestamp timestamp         = 3
      [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
  bytes signature = 4;
}
```

Reply:
Protobuf overhead?

We really need a way to compute that properly.

Reply:
> Protobuf overhead?

Yes, but it's unclear from where. `varint`s of 3 fields shouldn't take more than 1 byte because they are all very small.